.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Thanks for filing an issue! We appreciate the input.
+
+Before filing, a couple guidelines:
+
+- If this is a question about Protobuf, do not file an issue here. Consult the
+  Protobuf Language Guide at https://developers.google.com/protocol-buffers/docs/proto3
+  or ask your question on the Protobuf Mailing List at https://groups.google.com/forum/#!forum/protobuf.
+- If this is an issue with Protoc, do not file an issue here. File your issue
+  on the Protobuf repository at https://github.com/protocolbuffers/protobuf.
+- If this is a feature request, please provide a rough outline of the actual changes
+  you'd like to see with as many specifics as you can provide.
+- If this is a bug report, please provide specific steps to reproduce your issue.
+
+Please do not @ individual maintainers in the description! We appreciate it.
+
+If your issue doesn't meet the guidelines, we'll comment as such and close the issue - we don't
+mean to be impersonal, but we have to reduce the support burden and cannot be supporting all
+of Protobuf (in the case of Protobuf questions or Protoc issues) or have longstanding issues
```

Comment:
I would simply end this sentence at "but we have to reduce the support burden and cannot be supporting all of Protobuf." 

Then continue again at: "If the issue is a feature request ..."

Reply:
Done